syft/core/hooks/torch/hook.py

```diff
@@ -637,7 +503,7 @@ def new___repr__(self):
 
             tensor_type.__repr__ = new___repr__
 
-    def _hook_tensor_send_(hook_self, tensor_type):
+    def _hook_send_(hook_self, tensor_type):
```

Comment:
Needs a docstring.  Is this now generalized to include Variables and Parameters?  If so, the argument`tensor_type` needs to be renamed.

Reply:
+1